### ISSUE TITLE
Failure submitting job: unexpected keyword argument 'rwgtimeout'

### ISSUE THREAD

**Issue by rsdunlapiv** [esmf-org/esmf-extended-tests#2]
On Cheyenne:

```
python3 compweights.py --platform Cheyenne --esmfmkfile /glade/p/cesmdata/cseg/PROGS/esmf/8.1.0b26/intelmpi/2019.6.154/intel/19.0.5/lib/libO/Linux.intel.64.intelmpi.default/esmf.mk
```

results in:

```
Running 'compweights.py' with following input parameter values: 
--n =  4
--branch =  develop
--esmfmkfile =  /glade/p/cesmdata/cseg/PROGS/esmf/8.1.0b26/intelmpi/2019.6.154/intel/19.0.5/lib/libO/Linux.intel.64.intelmpi.default/esmf.mk
--platform =  Cheyenne
--gnu10 =  False
--rwgtimeout =  60
--debug_execdir =  

Skip ESMF build, esmf.mk provided.
Build test executable

Currently Loaded Modules:
  1) ncarenv/1.3   2) intel/19.0.5   3) impi/2019.6.154   4) pnetcdf/1.12.1   5) netcdf-mpi/4.7.3   6) pio/2.4.4

 


Submit the jobs for weight file generation: Wed, 02 Sep 2020 16:44:35
Traceback (most recent call last):
  File "/glade/scratch/dunlap/ryan/esmf-extended-tests/ESMF_CompWeights/src/run.py", line 214, in test
    do(df, EXECDIR, DATADIR, config, clickargs)
  File "/glade/scratch/dunlap/ryan/esmf-extended-tests/ESMF_CompWeights/src/run.py", line 150, in do
    job_threads.append(PropagatingThread(target=call_script, args=run_command, rwgtimeout=rwgtimeout, weights=weights, mb=""))
TypeError: __init__() got an unexpected keyword argument 'rwgtimeout'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "compweights.py", line 67, in cli
    EXECDIR = run.test(df, ESMFBINDIR, config, clickargs)
  File "/glade/scratch/dunlap/ryan/esmf-extended-tests/ESMF_CompWeights/src/run.py", line 219, in test
    raise RuntimeError("Error submitting the jobs.")
RuntimeError: Error submitting the jobs.
```

**Comment by rokuingh** [comment]
@rsdunlapiv, i mentioned in the demo that there was an issue with threading on Cheyenne. I made some mods today that I thought would work, but wasn't able to get all the way there. I will think on this and try again later.

**Comment by rokuingh** [comment]
resolved with [7399a9a](https://github.com/esmf-org/esmf-extended-tests/commit/7399a9ab6bad66d7d14c0c8c1794eae8ab3c8f86)
